Dockerfile

```diff
@@ -1,11 +1,14 @@
 # Read the Docs - Environment base
 FROM ubuntu:20.04
 LABEL mantainer="Read the Docs <support@readthedocs.com>"
+LABEL version="ubuntu-20.04-2021.09.23"
```

Comment:
Can we automate this with `date` or similar? Do we want to?

Reply:
I haven't try that. Honestly, I don't think `LABEL version` is strictly needed; but I thought it could be useful to use with `docker inspect` to know what's the version you have downloaded:

```
▶ docker inspect c54fcbdfb6c8 | jq '.[0].Config.Labels.version'
"ubuntu-20.04-2021.09.23"
```

It's a nice to have to me. If we can automate it, it would be 💯